### PR TITLE
Added Kwik source to animepahe

### PR DESCRIPTION
**Pull request by 29ayush**
I have added the kwik extractor.

Issues to yet work out.
1.) I have added kwik to only animepahe provider, it might be needed to be added to other existing providers.
2.) I don't know if the download will work with the aria for kwik (due to referer) [haven't checked yet].
3.) Same as 2 for mpv, the referer issue is it already fixed in mpv or it needs to be. [haven't checked.]

### CODE REVIEW

File: anime_downloader/extractors/kwik.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from bs4 import BeautifulSoup
+
+from anime_downloader.extractors.base_extractor import BaseExtractor
+from anime_downloader import session
+
+session = session.get_session()
+
+
+class Kwik(BaseExtractor):
+    '''Extracts video url from mp4upload embed pages, performs a request
```
Comment (ngomile) [inline anime_downloader/extractors/kwik.py]: This comment needs to be changed or omitted since it is the same as the MP4Upload one.

**Comment by ngomile** [comment]
If this is working properly then nice job, I love animepahe and this had to be done. Just make sure that everything else is working just by doing some requests to other sites. We don't have  proper tests sadly. Just gotta wait for @vn-ki.

File: anime_downloader/extractors/kwik.py (hunk 1 above)
Comment (29ayush) [reply anime_downloader/extractors/kwik.py]: Yup Thanks for review, I copied and modified the mp4upload some lingering comments persist. 
I will update

**Comment by 29ayush** [comment]
+------------------------------+----------------------------------+----------------------------------+---------------------------------+
| Provider  Original (Stable)  |               Dev                |         Dev + My Changes         |                                 |
+------------------------------+----------------------------------+----------------------------------+---------------------------------+
| 9 Anime                      |  Working                         |  Working                         |  Working                        |
| AnimePahe                    |  Not Working                     |  Not Working                     |  Working                        |
| twist.moe                    |  Not Working(Certificate Error)  |  Not Working(Certificate Error)  |  Not Working(Certificate Error) |
| KissAnime                    |  Working                         |  Working                         |  Working                        |
| Masterani.me                 |  Working                         |  Working                         |  Working                        |
| KissCartoon                  |  Not Working                     |  Not Working                     |  Not Working                    |
| GogoAnime                    |  Working                         |  Working                         |  Working                        |
+------------------------------+----------------------------------+----------------------------------+---------------------------------+

**Comment by rehtlaw** [comment]
Did you make sure to run the twist.moe tests with `--disable-ssl`

**Comment by 29ayush** [comment]
My bad twist.moe is working with `--disable-ssl`

**Comment by 29ayush** [comment]
> 
> 
> Plz also test the `-u` command and paste the url's in IDM .....it doesn't work and gave the error

It won't work with IDM -u yet, since referer tag is not added.

**Comment by vn-ki** [comment]
@29ayush If you can add aria2 referer, I can merge it right away. I don't think I'll find time for it anytime soon.

**Comment by 29ayush** [comment]
> 
> 
> @29ayush If you can add aria2 referer, I can merge it right away. I don't think I'll find time for it anytime soon.
 Ok I will add that by tonight I guess.

**Comment by 29ayush** [comment]
Sorry took more time then I said. I checked aria2 with other providers nothing has broken and it is now working with animepahe kwik.

**Comment by 29ayush** [comment]
@vn-ki Is there anything left for this pull?

**Comment by vn-ki** [comment]
Thanks @29ayush

**Pull request merged**